sprintvotingapp/sprint/views.py

```diff
@@ -0,0 +1,536 @@
+            return Response(
+                {
+                    "message": "no such sprint found",
+                },
+                status=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            )
+
+
+class VoteParameter(APIView):
+    @verify_token
+    def post(self, request):
+        """
+        For Adding Parameter for voting
+        :param request:
+        :return: Response
+        """
+        try:
+            user = User.objects.get(id=request.data.get("user_id"))
+            if user.is_superuser:
+                serializer = ParamSerializer(data=request.data)
+                serializer.is_valid(raise_exception=True)
+                serializer.save()
+                return Response({
+                    "message": "Parameter Added Successful",
+                    "data": serializer.data
+                },
+                    status=status.HTTP_201_CREATED)
+            return Response(
+                {
+                    "message": "Only admin or superuser can add  parameter"
+
+                }, status=status.HTTP_401_UNAUTHORIZED
+            )
+        except ValidationError:
+            logging.error("Validation failed")
+            return Response(
+                {
+                    "message": "validation failed"
+                },
+                status=status.HTTP_400_BAD_REQUEST)
+        except Exception as e:
+
+            return Response({
+                "message": "Error Occurred",
+                "error": str(e)
+            }, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    @verify_token
+    def get(self, request):
+        """
+        For Getting the list of all the parameter
+        :param request:
+        :return:
+        """
+        try:
+            parameters = Parameter.objects.all()
+            serializer = ParamSerializer(parameters, many=True)
+            return Response(
+                {
+                    "message": "Here your Parameters",
+                    "data": serializer.data
+                }, status=status.HTTP_200_OK
+            )
+        except ValidationError:
+            logging.error("Validation failed")
+            return Response(
+                {
+                    "message": "validation failed"
+                },
+                status=status.HTTP_400_BAD_REQUEST)
+        except Exception as e:
+            logging.error(e)
+            return Response(
+                {
+                    "message": "Error Occurred",
+                    "error": str(e),
+                },
+                status=status.HTTP_400_BAD_REQUEST)
+
+    @verify_token
+    def put(self, request, id):
+        """
+        USed for updating existing Parameter
+        :param request:
+        :param paramID: id of parameter you want to update
+        :return:
+        """
+        try:
+            user = User.objects.get(id=request.data.get("user_id"))
+            if user.is_superuser:
+                parameter = Parameter.objects.get(id=id)
```

Comment:
should check if parameter exists in db before proceeding

Reply:
done